### DISCUSSION TITLE
🖊️ Color "grey" is not accepted

### DISCUSSION THREAD

**Discussion by MarleenGilsing** [hedyorg/hedy#6003] [Language Ideas]
In the turtle 'gray' (American English) is accepted as a color,  but 'grey' isn't eventhough that's the correct Brittish spelling.

**Comment by MarleenGilsing** [comment]
While we're on it, you can't use the command 'colour' either, but that might be a bit more difficult to change?

**Comment by Felienne** [comment]
Hi @MarleenGilsing!

Not very hard to change, maybe a bit of an issue since in Python, these thing also will not work. (in fact changing colour is easier since we already support multiple versions of a keyword, but grey goes straight to Python :) )
